bundles/org.openhab.binding.metofficedatahub/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="metofficedatahub" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>MetOfficeDataHub Binding</name>
+	<description>This is the binding for the Met Office DataHub service.</description>
+	<connection>cloud</connection>
+
```

Comment:
Please add the countries tag
For reference: https://www.openhab.org/docs/developer/addons/addon.html#xml-structure-for-add-on-definitions

Reply:
I have added gb, as the met office is UK based. Looking at the website I believe this would be the primary area that it covers. It's unclear if it possibly extends further than this.